src/Components/Patient/PatientHome.tsx

```diff
@@ -1511,33 +1440,35 @@ export const PatientHome = (props: any) => {
           Consultation History
         </h2>
         {consultationList}
-        {!isConsultationLoading && totalConsultationCount > limit && (
-          <div className="mt-4 flex w-full justify-center">
-            <Pagination
-              cPage={currentConsultationPage}
-              defaultPerPage={limit}
-              data={{ totalCount: totalConsultationCount }}
-              onChange={handleConsultationPagination}
-            />
-          </div>
-        )}
+        {(!isConsultationLoading && consultationListData?.count) ||
+          (0 > limit && (
+            <div className="mt-4 flex w-full justify-center">
+              <Pagination
+                cPage={currentConsultationPage}
+                defaultPerPage={limit}
+                data={{ totalCount: consultationListData?.count || 0 }}
+                onChange={handleConsultationPagination}
+              />
+            </div>
+          ))}
```

Comment:
Same here, avoid unnecessary check for limit is positive. It's a hardcoded value anyways.